app/views/second_level_browse_page/_show_curated_accordion.html.erb

```diff
@@ -58,12 +72,21 @@
         dimension114: "0",
       }
     %>
+    <%
+      show_all_attributes = {
+        event_name: "select_content",
+        type: "accordion",
+        index: 0,
+        index_total: accordion_contents.count + 1,
+      }
+    %>
     <%= render "govuk_publishing_components/components/accordion", {
       track_show_all_clicks: true,
       track_sections: true,
       heading_level: 2,
       data_attributes_show_all: {
         "track-options": custom_dimensions.to_json,
+        "ga4": show_all_attributes.to_json
```

Comment:
I might have missed it, but does this file need the `gtm-click-tracking` module adding somewhere?